samples/net/stats/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(KCONFIG_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/Kconfig)
```

Comment:
I am not sure of the relevancy of that sample: the net shell module is already a good example on how to use the stats api.

Reply:
Sure, but it means then 2 places where maintenance will be needed when changing net stats (if that happens). Also I am not sure that many users will need to use that API, it's quite specific.